gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -126,27 +116,27 @@ const NxCardPage = () =>
                         liveExample={NxCardVerticalExample}
                         codeExamples={nxCardVerticalCode}
                         defaultCheckeredBackground={true}>
-      Cards shown in vertical orientation. While we have shown some content heavy examples it is recomended to keep the
-      content as minimal as possible. The orientation of the cards is determined by a class applied to their container.
+      Cards shown in a row layout. While we have shown some content heavy examples it is recomended to keep the
+      content as minimal as possible. The layout of the cards is determined by a class applied to their container.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Alternate Vertical Card"
+                        id="nx-card-vertical-alt-example"
+                        liveExample={NxCardVerticalAltExample}
+                        codeExamples={nxCardVerticalAltCode}
+                        defaultCheckeredBackground={true}>
+      This variation of the row layout has the callout and the description text side-by-side (when content allows).
```

Comment:
Is "(when content allows)" still true?

Reply:
> Is "(when content allows)" still true?

Nope: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/33340d58faa9277d46178f76ac6e5ce46738f502